web/src/main/java/com/futurewei/alcor/web/entity/dataplane/InternalPortEntity.java

```diff
@@ -15,24 +15,27 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.futurewei.alcor.web.entity.port.PortEntity;
 import com.futurewei.alcor.web.entity.route.RouteEntity;
-import com.futurewei.alcor.web.entity.vpc.VpcEntity;
 import lombok.Data;
 
+import java.util.ArrayList;
 import java.util.List;
-import java.util.Set;
 
 @Data
-public class InternalPortEntityNB extends PortEntity {
+public class InternalPortEntity extends PortEntity {
 
-  @JsonProperty("routes")
-  private List<RouteEntity> routes;
+    @JsonProperty("routes")
+    private List<RouteEntity> routes;
 
-  @JsonProperty("neighbor_info")
-  private List<NeighborInfo> neighborIps;
+    @JsonProperty("neighbor_info")
+    private List<NeighborInfo> neighborInfos;
 
-  @JsonProperty("binding_host_ip")
-  private String bindingHostIP;
+    @JsonProperty("binding_host_ip")
+    private String bindingHostIP;
 
-  private Set<InternalSubnetEntityNB> subnetEntities;
```

Comment:
@chenpiaoping Check a few latest commits that were not included here: https://github.com/futurewei-cloud/alcor/pull/244/commits, particularly this one https://github.com/futurewei-cloud/alcor/pull/244/commits/cdbd96e349b92732bad75f656d69fadb19529695

Reply:
Why does InternalPortEntity need the fields vpcEntities and subnetEntities？Don't they conflict with vpcEntities and subnetEntities in NetworkConfiguration?

Reply:
Good question. Asked the same question when I saw them at the first place. Those two fields are internal fields which are not exposed to other services (port manager don't have to pass to DPM. DPM uses those two fields for internal bookkeeping purpose. @haboy52581